examples/confidential_computing/main.tf

```diff
@@ -65,7 +65,8 @@ resource "google_kms_crypto_key_iam_binding" "crypto_key" {
 }
 
 module "instance_template" {
-  source = "../../modules/instance_template"
+  source = "terraform-google-modules/vm/google//modules/instance_template"
```

Comment:
nit: // -> /

Reply:
Hi @iennae - I believe the `//` is required for sub-modules when using the registry?

https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

Reply:
oh woah! TIL! Thank you!!